utils/compare_results.py

```diff
@@ -46,6 +46,32 @@ def get_wining_status(cls, current_status: str, proposed: str) -> str:
         return proposed
 
 
+class Comparison:
+    same_status = list()
+    missing_in_target = list()
+    different_results = dict()
```

Comment:
Those are declared as class variables - I suggest to move that under `__init__` in a similar way as e.g. `base_results`. The [dataclass](https://docs.python.org/3/library/dataclasses.html) syntax is similar, but data classes are a relatively new Python feature, so we can't use it yet.